markdown/docs/cronjobblog.mdx

```diff
@@ -0,0 +1,150 @@
+---
+pubDate: 'Aug 19 25'
+title: 'The Cronjob Conundrum'
+description: 'Instrumenting Cronjobs with Odigos: Part of the Blind Spot Series'
+metadata: 'Instrumenting Cronjobs with zero code changes and no restarts with Odigos'
+tags: [odigos, opentelemetry, cronjobs, Instrumentation, Traces, Automation, Blind Spot Series]
+category: 'Odigos'
+author: 'Will Searle'
+authorImage: ''
+image: '/images/blog/cronjobblog/CoverImageResize.png'
+webCoverImage: '/images/blog/cronjobblog/CoverImage1.png'
+mobileCoverImage: '/images/blog/cronjobblog/CoverImage1.png'
+---
+
+## Blind Spot Series: The CronJob Conundrum
+
+### What are Kubernetes Cronjobs, and Why do They Matter?
+
+Kubernetes CronJobs let you schedule recurring tasks such as backups,
+batch processing, or data syncs.  They are basically the
+“set-it-and-forget-it” automation layer of modern Kubernetes
+infrastructure. They run containers on a time-based schedule making them
+essential for repetitive quick tasks or relaying data or messages from
+many different services.
+
+Despite their simplicity, CronJobs are critical to the reliability and
+performance of many systems.  When CronJobs fail, it can lead to silent
+data loss, corrupted downstream services, or even missed SLAs.
+
+### The Conundrum...
+
+Instrumentation of CronJobs can be challenging due to their
+architecture and nature of their workload. Their ephemeral runtime
+often concludes before observability pipelines can capture telemetry, 
```

Comment:
this is a huge point for cronjobs (and serverless, which cronjobs are kind of similar to in a way). the overhead of observability eats into the ephemerality of the cronjob which is the entire point 👍